interviewing/readme.md

```diff
@@ -0,0 +1,66 @@
+# Interviewing 🧑‍💻
+
+Acing job interviews is a skill that can be learned like any other. Today we will pair up to practice interviewing.
+
+## Objective 💬
+
+At the end of this workshop, participants can:
+
+- [] Introduce themselves in 2 minutes
+- [] Answer soft skill interview questions using the STAR method
+- [] Prepare a strategy to get ready for  technical interviews
+
+## Setup 🌼
+
+- Pair up into groups of 2-3
+
+## Stage 1: Introduce Yourself 🙆
+
+In almost every interview, the interviewer will ask you to introduce yourself. The goal is to give the interviewer a snapshot of who you are, your professional background, and why you're a good fit for the role. Take 5 mins to craft a compelling narrative. 
+
+Some tips:
+- Your response should be 1-2 minutes
+- Try to structure your response, as an example the Present-Past-Future model:
+    - Present: Start by briefly mentioning your current role or what you are doing at the moment (education, job, etc.), and give an overview of your responsibilities.
+    - Past: Mention your relevant experience, skills, or education. Focus on achievements and how they shaped your current professional identity.
+    - Future: Highlight what you want to do next and why you’re excited about this opportunity.
+- Try to be enthusiastic, let your personality and confidence shine
+- Do not overwhelm your interviewer with too much information
+- Allow 1-2 minutes for follow-up questions
+
+After the 5 minutes are up take turns practicing your introductions. While the other person is introducing themselves, put yourself in an interviewers shoes. Share what the other person did well and what they could improve.
+
+## Step 2: The Soft Skills Interview 🥎 
+
+Soft skill interview questions are typically asked at some point during the interview process. 
+A common technique to answer these questions is to use the [STAR method](https://ca.indeed.com/career-advice/interviewing/star-interview-method) which stands for Situation, Task, Action and Result.
+
+“Can you tell me about a time when you had to work in a team and faced a challenge?”
+
+- <strong> Situation </strong>: "During my final group project, we had to develop a mobile app for managing student assignments. Midway through the project, we hit a roadblock because two teammates disagreed on which framework to use—one preferred React Native, while the other wanted to stick with Flutter. This was causing delays in our progress.”
```

Comment:
```suggestion
- <strong> Situation </strong>: "During my final group project, we had to develop a mobile app for managing student assignments. Midway through the project, we hit a roadblock because two teammates disagreed on which framework to use: one preferred React Native, while the other wanted to stick with Flutter. This was causing delays in our progress.”
```